string name
string parent
geometry_msgs/Pose pose
---
int32 error_code
